src/wp-admin/includes/ajax-actions.php

```diff
@@ -4035,7 +4035,7 @@ function wp_ajax_crop_image() {
 			}
 
 			/** This filter is documented in wp-admin/includes/class-custom-image-header.php */
-			$cropped    = apply_filters( 'wp_create_file_in_uploads', $cropped, $attachment_id ); // For replication.
+			$cropped = apply_filters( 'wp_create_file_in_uploads', $cropped, $attachment_id ); // For replication.
 
```

Comment:
Remove changes to spacing. I removed the spaces myself, but I hadn't noticed that the actual change was to add a line break between these lines of code, and it would be better to leave it unchanged.